cloud-computing/gramine/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # FIXME: use tagged version when released
-FROM docker-regis.iex.ec/iexec-graphene-base:0.7.0
+FROM docker-regis.iex.ec/iexec-graphene-base:0.8.0
 
 RUN  apt-get install -y python3 \
```

Comment:
Would it be possible to clean apt cache here before merge ?

Reply:
Thanks

Reply:
You'r welcome, thanks :)